src/osdatahub/DownloadsAPI/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .data_package import *
```

Comment:
Might be better to be more explicit here - assuming you want to import the main class in each file?